applications/app/views/signup/newsletterContent.scala.html

```diff
@@ -106,15 +106,13 @@ <h3 class="signup-confirmation--success">Check your Inbox!</h3>
             <div class="newsletters-heading">
                 Guardian newsletters: sign&nbsp;up
             </div>
-            <div class="newsletters-heading__manage-button-wrapper">
-                <a
-                data-link-name="all-newsletters : manage-emails"
-                href=@manageEmailsUrl>
-                    <button class="newsletters-heading__manage-button">
-                        <span>Manage my newsletters</span>
-                    </button>
-                </a>
-            </div>
+            <a data-link-name="all-newsletters : manage-emails" href="@manageEmailsUrl">
+                <div class="newsletters-heading__manage-button">
+                    <span class="">
```

Comment:
you might want to drop then empty class attribute as well

Reply:
My kingdom for a HTML linter (inside IntelliJ)